sky/authentication.py

```diff
@@ -195,7 +198,37 @@ def setup_gcp_authentication(config):
                         credentials=None,
                         cache_discovery=False)
     user = config['auth']['ssh_user']
-    project = compute.projects().get(project=project_id).execute()
+
+    try:
+        project = compute.projects().get(project=project_id).execute()
+    except googleapiclient.errors.HttpError as e:
```

Comment:
do we want to add another `except` at the end to catch other unexpected errors?

Reply:
sure that makes sense. was thinking about printing some additional msgs but I don't think it's necessary as users can't easily fix this themselves.